freqtrade/rpc/telegram.py

```diff
@@ -840,14 +841,20 @@ def _balance(self, update: Update, context: CallbackContext) -> None:
                             f"*{curr['currency']}:*\n"
                             f"\t`Available: {curr['free']:.8f}`\n"
                             f"\t`Balance: {curr['balance']:.8f}`\n"
-                            f"\t`Pending: {curr['used']:.8f}`\n"
-                            f"\t`Est. {curr['stake']}: "
-                            f"{round_coin_value(curr['est_stake'], curr['stake'], False)}`\n")
+                            f"\t`Pending: {curr['used']:.8f}`\n")
+                        if curr['stake'] == curr['stake']:
+                            reserved = self._rpc._freqtrade.wallets.get_reserved_stake_amount()
+                            curr_output += f"\t`Reserved: {reserved}`\n"
```

Comment:
I think we'd want to have an update to `telegram.md` for this as well - explaining what "Reserved" actually means.
it's not 100% clear from the call alone (i've had to dig through the code to figure out what you actually mean).